src/scripts/cp.js

```diff
@@ -229,7 +230,7 @@ CoursePresentation.prototype.attach = function ($container) {
           '     <div class="h5p-slides-wrapper"></div>' +
           '    </div>' +
           '  </div>' +
-          '  <nav class="h5p-cp-navigation">' +
+          '  <nav class="h5p-cp-navigation" aria-labelledby="' + this.l10n.slideshowNavigationLabel + '">' +
```

Comment:
`aria-labelledby` expects the id of the element containing the corresponding label, not the actual label. Consider using `aria-label` instead